plexpy/versioncheck.py

```diff
@@ -101,7 +101,7 @@ def get_version():
         else:
             cur_commit_hash = str(output)
 
-        if not re.match('^[a-z0-9]+$', cur_commit_hash):
+        if not re.match('^[a-z0-9]+$', str(cur_commit_hash)):
             logger.error('Output does not look like a hash, not using it.')
             cur_commit_hash = None
```

Comment:
I think the `if` block just needs to be indented one more level.

```suggestion
            if not re.match('^[a-z0-9]+$', cur_commit_hash):
                logger.error('Output does not look like a hash, not using it.')
                cur_commit_hash = None
```

Reply:
That makes sense. I'll add this change to the PR.